web-components/src/components/usa-banner/index.js

```diff
@@ -76,9 +218,15 @@ export class UsaBanner extends LitElement {
                 />
                 <div class="usa-media-block__body">
                   <p>
-                    <strong>Official websites use .gov</strong><br />A
-                    <strong>.gov</strong> website belongs to an official
-                    government organization in the United States.
+                    <strong>
+                      <slot name="domain-heading">
+                        ${domain.heading} .${tld}
+                      </slot>
+                    </strong>
+                    <br />
+                    <slot name="domain-text">
+                      ${domain.text1} <strong>.${tld}</strong> ${domain.text2}
+                    </slot>
```

Comment:
Added in b580613.  Not seeing a big benefit, but trying nonetheless because it could increase template readability.


The only small downside is now I have to pass the `TLD` to each template.

```js
${this.domainTemplate(tld)}
```

I'm sure there's a cleaner way to fallback to default value if user doesn't pass anything. Will look to iterate on this in future re-work for banner.